package/package.json

```diff
@@ -4,6 +4,7 @@
 	"description": "Isomorphic secure-random library, demonstrating `exports` use with Workers",
 	"exports": {
 		"node": "./src/node.js",
-		"workerd": "./src/workerd.mjs"
+		"workerd": "./src/workerd.mjs",
+		"worker": "./src/workerd.mjs"
```

Comment:
What is this about?
Can it not deal with `workerd`?

Reply:
`workerd` is a cloudflare-only concept that other bundlers aren't aware of. In general it is good to be as explicit as possible. For instance when wrangler looks at the exports map, it will look for anything that matches its category. That would be something like looking for "workerd", "worker", "browser", and then "main" as a last resort. So any of these is actually valid. tl;dr; This is also what we would expose in our own exports map.

Reply:
Interesting, this example is straight from Cloudflare - but of course their wrangler with esbuild handled `workerd` fine.

Reply:
I tried to find a way for webpack to understand `workerd` as a target, it seems it only knows `worker`.